config/kubernetes/sig-testing/teams.yaml

```diff
@@ -37,3 +37,61 @@ teams:
     - cjwagner
     - mkumatag
     privacy: closed
+  test-infra-admins:
+    description: Administrators to test-infra repo
```

Comment:
Should I be in this one?

Reply:
this was just a move of the existing lists above to scope them into the SIG. there should be a follow up to clean up the lists.

Reply:
all of these teams are copied from what is in github, and are therefore almost undoubtedly out of date.

Reply:
SGTM